docs/releases/2.2.4.rst

```diff
@@ -0,0 +1,66 @@
+Translate Toolkit 2.2.4
+***********************
+
+*Released on 31 July 2017*
+
+This release contains improvements and bug fixes.
+
+
+Major changes
```

Comment:
I think previous notes dropped Major.  These are not major changes.  They're just 'Summary of Changes'